bulk_add/Makefile

```diff
@@ -0,0 +1,17 @@
+BUILD_NUMBER ?= 0
+BMO_VERSION = 0.5.1
+IMG_TAG = v$(BMO_VERSION)-$(BUILD_NUMBER)
```

Comment:
change = to ?=  so we can override it in TC build

Reply:
Done.